### PR TITLE
Gamess thermochemistry

### PR DESCRIPTION
**Pull request by markperri**
I added parsing for Enthalpy, Entropy, Free Energy, Temperature, and Pressure for GAMESS files.  GAMESS has the ability to compute these at multiple temperatures, but I think that cclib only has room for one.  I didn't do anything special for that, so it will probably replace the values while it parses.

I added them to the last value of scfenergies, but I'm not sure if I should check for the existence of ccenergies or mpenergies?

### CODE REVIEW

File: cclib/parser/gamessparser.py [hunk 1]
```diff
@@ -1462,6 +1462,77 @@ def extract(self, inputfile, line):
...
+            match = re.search(r"P=(.*)PASCAL.", line)
+            if match:
+                self.set_attribute('pressure', float(match.group(1))/1.01325e5)
+
+        if "KCAL/MOL  KCAL/MOL  KCAL/MOL CAL/MOL-K CAL/MOL-K CAL/MOL-K" in line:
+            line = next(inputfile) #ELEC
+            line = next(inputfile) #TRANS
+            line = next(inputfile) #ROT
+            line = next(inputfile) #VIB
+            line = next(inputfile) #TOTAL
```
Comment (shivupa) [inline cclib/parser/gamessparser.py]: Can you make this : `self.skip_lines(inputfile, ["ELEC", "TRANS", "ROT", "VIB", "TOTAL"])` If I am understanding #765 that should be right. @langner and @berquist might have to check me.

**Comment by shivupa** [comment]
Also I like how this looks, but could you add a test. In `testdata`, [line 306](https://github.com/cclib/cclib/blob/572fab907a7fb2aaf0bd26e1eeb7e319c4bd354e/test/testdata#L306) and [line 307](https://github.com/cclib/cclib/blob/572fab907a7fb2aaf0bd26e1eeb7e319c4bd354e/test/testdata#L307) say that files [dvb ir 2017](https://github.com/cclib/cclib/blob/master/data/GAMESS/basicGAMESS-US2017/dvb_ir.out) and [dvb ir 2018](https://github.com/cclib/cclib/tree/master/data/GAMESS/basicGAMESS-US2018) are being tested by [this class](https://github.com/cclib/cclib/blob/master/test/data/testvib.py#L171). Could that class be modified to test the data this parses: `enthalpy`, `entropy` and `freeenergy` in the same way that the [Molcas](https://github.com/cclib/cclib/blob/572fab907a7fb2aaf0bd26e1eeb7e319c4bd354e/test/data/testvib.py#L79) or [Orca](https://github.com/cclib/cclib/blob/572fab907a7fb2aaf0bd26e1eeb7e319c4bd354e/test/data/testvib.py#L109) tests are done? Let me know if it is unclear what needs to be done!

**Comment by markperri** [comment]
Thanks, I wanted to add a test, but I didn't quite know where to put it.

**Comment by langner** [comment]
Thanks for this! I created #779 for the muti-temperature feature.

**Comment by markperri** [comment]
Thanks!  Let me know if you need any other GAMESS stuff.

**Pull request merged**